src/App.svelte

```diff
@@ -269,6 +271,17 @@
 		height: 19rem;
 	}
 
+	.about-image {
+		height: 32rem;
+		object-fit: cover;
+	}
+
+	.register-button {
+ 		width: 15rem;
+  		height: 3rem;
+		background-color: var(--header);
+		color: white;
```

Comment:
the text color on the button should be the same as the background

Reply:
fixed this sorry I thought I fixed it before